accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -89,6 +89,17 @@ Windows-specific:
 
 ### System
 
+* **ArgIterator**
```

Comment:
Should we mark the entire type?

Reply:
Note that this type works fine on Mono.

There is no fundamental reason why this type cannot work xplat. We just have not done the work to make it work in CoreCLR.

Reply:
In that case we shouldn't mark it.

Reply:
Actually, @fxdc thinks we he API annotation should be reflective of the current release. If that changes later, we can remove the annotation.

Reply:
We are shipping Mono in the current release.

Reply:
Oh I see. In that case we should not mark it then because we err on the side of false negatives, rater than false positives.